frontend/src/views/material/MaterialOverview.vue

```diff
@@ -0,0 +1,105 @@
+              <v-icon>mdi-dots-vertical</v-icon>
+            </v-btn>
+          </template>
+          <v-list class="py-0">
+            <DialogMaterialListCreate v-if="!isGuest" :camp="camp()">
+              <template #activator="{ attrs, on }">
+                <v-list-item v-bind="attrs" v-on="on">
+                  <v-list-item-icon>
+                    <v-icon>mdi-plus</v-icon>
+                  </v-list-item-icon>
+                  <v-list-item-content
+                    >{{ $tc('views.material.materialOverview.createNewList') }}
+                  </v-list-item-content>
+                </v-list-item>
+              </template>
+            </DialogMaterialListCreate>
+            <v-list-item @click="downloadXlsx">
+              <v-list-item-icon>
+                <v-icon>mdi-microsoft-excel</v-icon>
+              </v-list-item-icon>
+              <v-list-item-content
+                >{{ $tc('views.material.materialOverview.download') }}
+              </v-list-item-content>
+            </v-list-item>
+          </v-list>
+        </v-menu>
+      </template>
+      <v-expansion-panels
+        v-if="collection.length > 1"
+        v-model="openPeriods"
+        multiple
+        flat
+        accordion
+      >
+        <PeriodMaterialLists
+          v-for="{ period, materialItems } in collection"
+          :key="period._meta.self"
+          :period="period"
+          :material-item-collection="materialItems"
+        />
+      </v-expansion-panels>
+      <v-card-text v-else-if="collection.length === 1">
+        <MaterialTable
+          v-for="{ period, materialItems } in collection"
+          :key="period._meta.self"
+          :camp="camp()"
+          :material-item-collection="materialItems"
+          :period="period"
+          :disabled="!isContributor"
+        />
+      </v-card-text>
+    </content-card>
+  </v-container>
+</template>
+
+<script>
+import ContentCard from '@/components/layout/ContentCard.vue'
+import MaterialTable from '@/components/material/MaterialTable.vue'
+import PeriodMaterialLists from '@/components/material/PeriodMaterialLists.vue'
+import DialogMaterialListCreate from '@/components/campAdmin/DialogMaterialListCreate.vue'
+import { campRoleMixin } from '@/mixins/campRoleMixin.js'
+import { useMaterialViewHelper } from '@/components/material/useMaterialViewHelper.js'
+import { apiStore } from '@/plugins/store/index.js'
+
+export default {
+  name: 'MaterialOverview',
+  components: {
+    ContentCard,
+    DialogMaterialListCreate,
+    MaterialTable,
+    PeriodMaterialLists,
+  },
+  mixins: [campRoleMixin],
+  props: {
+    camp: { type: Function, required: true },
+  },
+  setup({ camp }) {
+    return useMaterialViewHelper(camp.call({ api: { get: apiStore.get } }))
+  },
+  computed: {
+    collection() {
+      return this.camp()
+        .periods()
+        .items.map((period) => ({
+          period,
+          materialItems: apiStore.get().materialItems({
```

Comment:
On the deployment, I see every call to material_items endpoint happening twice. I could not reproduce on localhost (maybe a timing issue?), but [on the deployment](https://pr3797.ecamp3.ch/camps/6973c230d6b1/Harry-Potter-Lager/material/all?isDetail=true) it looks as following for me:

<img width="1158" alt="image" src="https://github.com/ecamp/ecamp3/assets/449555/8fffd09a-35bd-446a-acfd-052ce982246c">

2 hypothesis of what's happening:
- Vue waits for material items to load. Afterwards, within PeriodMaterialLists.vue a reload is triggered.
- It's related to https://github.com/ecamp/ecamp3/issues/3308 (because one call is with `/periods/903bafdeed45` and the other with `/api/periods/903bafdeed45`)

Reply:
Fixed this in 4d842ab. It was the PeriodMaterialLists that triggered it a second time. I reloading get's done by the composition `useMaterialViewHelper` on mount.